portfolio.html

```diff
@@ -54,5 +54,113 @@ <h2>I am a Software Developer</h2>
         </div>
       </div>
     </section>
+
+    <section class="works">
+      <div id="set1">
+        <h1>My Recent Works</h1>
+      </div>
+      <div>
+        <hr class="solid" />
+      </div>
+      <div>
+        <img
+          class="set-image"
+          src="/images/ImgPlaceholder.png"
+          alt="works image"
+        />
+      </div>
+      <div id="set2">
+        <h2>Multi-Post Stories</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div id="lang">
+        <img src="/images/Icons/languages/css.png" alt="csslang" />
+        <img src="/images/Icons/languages/html.png" alt="htmllang" />
+        <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
+        <img src="/images/Icons/languages/ruby.png" alt="rubylang" />
+      </div>
```

Comment:
Kindly wrap this category section in a 'ul` tag and put each element as `li` tag as stated in the project requirements, so your code will be more professional